core/src/main/java/com/linecorp/armeria/common/stream/StreamWriter.java

```diff
@@ -133,4 +135,34 @@ default boolean tryWrite(Supplier<? extends T> o) {
      * signal that the {@link Subscriber} did not consume the stream completely.
      */
     void close(Throwable cause);
+
+    /**
+     * Adapts this {@link StreamWriter} to {@link OutputStream}.
+     *
+     * <p>For example:<pre>{@code
+     * DefaultStreamMessage<String> writer = new DefaultStreamMessage<>();
+     * int maxBufferSize = 128;
+     * OutputStream outputStream = writer.toOutputStream(x -> {
+     *     byte[] data = new byte[x.byteBuf().readableBytes()];
+     *     x.byteBuf().readBytes(data);
+     *     return new String(data);
+     * }, maxBufferSize);
+     *
+     * List<String> strings = List.of("foo", "bar", "baz");
+     * for (String string : strings) {
+     *     for (byte b : string.getBytes()) {
+     *         outputStream.write(b);
+     *     }
+     *     outputStream.flush();
+     * }
+     * outputStream.close();
+     *
+     * assert writer.collect().join().equals(List.of("foo", "bar", "baz"));
+     * }</pre>
+     */
+    default OutputStream toOutputStream(Function<? super HttpData, ? extends T> httpDataConverter,
```

Comment:
How about adding `toOutputStream(Function<? super HttpData, ? extends T> httpDataConverter)` with the default buffer size(8192)?

Reply:
✅ updated!